service-admin/composer.json

```diff
@@ -67,7 +61,9 @@
         "phpunit/phpunit": "^9.5.10",
         "roave/security-advisories": "dev-master",
         "squizlabs/php_codesniffer": "^3.6.1",
-        "vimeo/psalm": "^4.23"
+        "vimeo/psalm": "^4.23",
+        "symfony/validator": "^4.2",
+        "egulias/email-validator": "~1.2"
```

Comment:
I thought laminas did the email validation? It probably could (but that might be for another ticket).

Reply:
OK removed because turned out not needed